### PR TITLE
unifies attention grabbing messages

### PR DESCRIPTION
**Pull request by EverlastingBugstopper**
This PR replaces the `message::big_info` function with `message::boxx` which uses a [Boxx](https://docs.rs/boxx) with a project-wide style. All message boxes using this function will be configured the same way, with a rounded, cyan border and a 1 character margin.

```
$ wrangler preview
💁  JavaScript project found. Skipping unnecessary build!

                       ╭─────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
                       │                                                                                                                 │
                       │                                       You haven't run `wrangler config`.                                        │
                       │                                                                                                                 │
                       │                                      Please run `wrangler config` or visit                                      │
                       │      https://developers.cloudflare.com/workers/tooling/wrangler/configuration/#using-environment-variables      │
                       │                               for info on configuring with environment variables.                               │
                       │                                                                                                                 │
                       ╰─────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯

💁  Running preview without authentication.
👷  Your Worker responded with: you did a new url https://example.com/
```

```
$ wrangler dev

                                ╭────────────────────────────────────────────────────────────────────────────────────────────────╮
                                │                                                                                                │
                                │        wrangler dev is currently unstable and there are likely to be breaking changes!         │
                                │      For this reason, we cannot yet recommend using wrangler dev for integration testing.      │
                                │                                                                                                │
                                │      Please submit any feedback here: https://github.com/cloudflare/wrangler/issues/1047       │
                                │                                                                                                │
                                ╰────────────────────────────────────────────────────────────────────────────────────────────────╯
```

```
$ wrangler config

                                    ╭────────────────────────────────────────────────────────────────────────────────────────╮
                                    │                                                                                        │
                                    │      To find your API token, go to https://dash.cloudflare.com/profile/api-tokens      │
                                    │              and create it using the "Edit Cloudflare Workers" template.               │
                                    │                                                                                        │
                                    │          If you are trying to use your Global API Key instead of an API Token          │
                                    │                  (Not Recommended), run "wrangler config --api-key".                   │
                                    │                                                                                        │
                                    ╰────────────────────────────────────────────────────────────────────────────────────────╯

Enter API token:
```

```
$ wrangler config --api-key

                              ╭───────────────────────────────────────────────────────────────────────────────────────────────────╮
                              │                                                                                                   │
                              │                           We don't recommend using your Global API Key!                           │
                              │                            Please consider using an API Token instead.                            │
                              │                                                                                                   │
                              │      https://support.cloudflare.com/hc/en-us/articles/200167836-Managing-API-Tokens-and-Keys      │
                              │                                                                                                   │
                              ╰───────────────────────────────────────────────────────────────────────────────────────────────────╯

Enter email:
```

### CODE REVIEW

File: src/commands/dev/mod.rs [hunk 1]
```diff
@@ -42,7 +43,11 @@ pub fn dev(
     ip: Option<&str>,
     verbose: bool,
 ) -> Result<(), failure::Error> {
-    message_box::dev_alpha_warning();
+    let wrangler_dev_msg = style("`wrangler dev`").yellow().bold();
+    let feedback_url = style("https://github.com/cloudflare/wrangler/issues/1047")
+        .blue()
+        .bold();
+    message::boxx(&format!("{0} is currently unstable and there are likely to be breaking changes!\nFor this reason, we cannot yet recommend using {0} for integration testing.\n\nPlease submit any feedback here: {1}", wrangler_dev_msg, feedback_url));
```
Comment (ashleymichal) [inline src/commands/dev/mod.rs]: you talked about naming this "billboard"?

Comment (ashleymichal) [inline src/commands/dev/mod.rs]: you had talked about naming this "billboard"?

**Comment by victoriabernard92** [comment]
Adding Nena and John on this as I think they'd have useful input on the design and the copy

File: src/commands/dev/mod.rs (hunk 1 above)
Comment (EverlastingBugstopper) [reply src/commands/dev/mod.rs]: yah i'll rename it client side for now and then i'm going to rename the actual crate sometime later and then we can update that in a subsequent pr

**Comment by EverlastingBugstopper** [comment]
cool! I'm going to merge this now and if there are further improvements we can address them then 😄

File: src/main.rs [hunk 2]
```diff
@@ -508,20 +509,30 @@ fn run() -> Result<(), failure::Error> {
...
     if let Some(matches) = matches.subcommand_matches("config") {
         // If api-key flag isn't present, use the default auth option (API token)
         let default = !matches.is_present("api-key");
 
         let user: GlobalUser = if default {
             // API Tokens are the default
-            message::big_info("To find your API token, go to https://dash.cloudflare.com/profile/api-tokens\n\tand create it using the \"Edit Cloudflare Workers\" template");
-            message::big_info("If you are trying to use your Global API Key instead of an API Token\n\t(Not Recommended), run \"wrangler config --api-key\".\n");
-            let api_token: String = interactive::get_user_input("Enter API token: ");
+            message::billboard(&format!("To find your API Token, go to {}\nand create it using the \"Edit Cloudflare Workers\" template.\n\nIf you are trying to use your Global API Key instead of an API Token\n{}, run {}.", not_recommended_msg, recommended_cmd_msg, api_token_url));
```
Comment (porty) [inline src/main.rs]: The arguments seem to be out of order: 
image

Comment (EverlastingBugstopper) [reply src/main.rs]: 👀 good catch! thanks

**Pull request merged**